src/screens/CreatePost/index.js

```diff
@@ -610,6 +622,7 @@ const CreatePost = () => {
 
   const isPollButtonDisabled = () => getReducedPoll().length < 2;
   const isButtonDisabled = () => {
+    if (loadingPost) return true;
     if (isPollShown) return isPollButtonDisabled();
     return false;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The update to the `isButtonDisabled` function is a good practice. It now considers the `loadingPost` state, disabling the button when a post is being created. This prevents multiple submissions by the user.

```javascript
const isButtonDisabled = () => {
  if (loadingPost) return true;
  if (isPollShown) return isPollButtonDisabled();
  return false;
};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->